src/pages/studyView/table/GenePanelModal.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import { Modal, Button } from "react-bootstrap";
+import { GenePanelToGene, GenePanel } from "shared/api/generated/CBioPortalAPIInternal";
+
+interface IGeneModalProps {
+    genes: GenePanelToGene[];
+    panelName: string;
+    show: boolean;
+    hide: () => void;
+}
+
+interface IGenePanelTooltipProps {
+    genePanels: GenePanel[];
+    toggleModal: (panelName: string, genes: GenePanelToGene[]) => void;
+}
+
+export const GenePanelList: React.FunctionComponent<IGenePanelTooltipProps> = ({
+    genePanels,
+    toggleModal
+}) => {
+    if (genePanels.length > 0) {
+        return (
+            <span>
+                <br />
```

Comment:
what's reason for this BR?

Reply:
for spacing, its use here is equivalent to a newline